mbed_host_tests/tags

```diff
@@ -0,0 +1,402 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
This looks like ctags output or something. Is this file needed?

Reply:
Definitely not, my bad